repo/build.gradle

```diff
@@ -33,8 +33,8 @@ repositories {
 
     maven {
```

Comment:
I'd remove this entire section including the private URL if you're wanting to convert this to community maven.